conduction-design-tokens/src/component/conduction/alert.tokens.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
As discussed in our call: all the tokens that are within the `/conduction` directory should be prefixed with "conduction" and not "utrecht".